test/database/list.mjs

```diff
@@ -0,0 +1,77 @@
+//@ts-check
+
+import { expect } from "chai";
+import { ServiceError } from "fauna";
+import sinon from "sinon";
+
+import { run } from "../../src/cli.mjs";
+import { setupTestContainer as setupContainer } from "../../src/config/setup-test-container.mjs";
+
+describe("database list", () => {
```

Comment:
what's the plan to test we get the output we actually we want?

Is that being saved for the integration test story?